iamy/cfn.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/cloudformation/cloudformationiface"
 )
 
-var cfnResourceRegexp = regexp.MustCompile(`-[A-Z0-9]{10,20}$`)
+var cfnResourceRegexp = regexp.MustCompile(`^(?=.{10,20})(?=.*[A-Z])(?=.*[0-9A-Z]).*$`)
```

Comment:
Shall we add an additional test? https://github.com/envato/iamy/blob/e13a87f129bef1aa17fc2c776e09c49b32713f55/iamy/cfn_test.go#L30-L32

Reply:
Yep!  Doing that now.

Reply:
521f98d